src/librustc_codegen_llvm/back/lto.rs

```diff
@@ -517,11 +517,20 @@ fn thin_lto(
 
                 let prev_imports = prev_import_map.modules_imported_by(module_name);
                 let curr_imports = curr_import_map.modules_imported_by(module_name);
+                let prev_exports = prev_import_map.modules_exported_by(module_name);
+                let curr_exports = curr_import_map.modules_exported_by(module_name);
                 let imports_all_green = curr_imports
                     .iter()
                     .all(|imported_module| green_modules.contains_key(imported_module));
+                let exports_all_green = curr_exports
```

Comment:
Could you add a comment here linking to https://github.com/rust-lang/rust/issues/69798? When I was investigating the issue, it wasn't obvious to me that the *exports* of a module could affect its codegen.

Reply:
will do!